packages/dashboard/src/stores/config-store.ts

```diff
@@ -7,10 +7,37 @@ export default class ConfigStore extends BaseStore {
     public asset_path = '';
     public routes: TRoutesProps = {
         home: '/',
+        my_apps: '/my-apps',
         explore: '/explore',
         about_us: '/about-us',
         resources: '/resources',
-        platform_dmt5_synthetic: '/platforms/dmt5_synthetic',
+
+        market_commodities: '/markets/commodities',
+        market_forex: '/markets/forex',
+        market_stock: '/markets/stock',
+        market_synthetic: '/markets/synthetic',
+        markets: '/markets',
+
+        platform_binary_bot: '/platforms/binary-bot',
+        platform_dbot: '/platforms/dbot',
+        platform_dmt5: '/platforms/dmt5',
+        platform_dmt5_financial: '/platforms/dmt5-financial',
+        platform_dmt5_financial_stp: '/platforms/dmt5-financial-stp',
+        platform_dmt5_synthetic: '/platforms/dmt5-synthetic',
+        platform_dtrader: '/platforms/dtrader',
+        platform_smarttrader: '/platforms/smarttrader',
+        platforms: '/platforms',
+
+        trade_type_cfds: '/trade-types/cdfs',
```

Comment:
```suggestion
        trade_type_cfds: '/trade-types/cfds',
```

Reply:
Thanks, my search panel stopped working in vscode, so It didn't show me this